From: Chris Lamb <lamby@debian.org>
Date: Fri, 12 May 2017 10:49:00 +0200
Subject: remove-timestamp

Capturing the current time during builds prevent packages from building
reproducibly. One can already know when a manpage has been generated by looking
at the file creation time.
---
 Transpec/docbook-to-man.ts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Transpec/docbook-to-man.ts b/Transpec/docbook-to-man.ts
index c466a8f..98803e7 100644
--- a/Transpec/docbook-to-man.ts
+++ b/Transpec/docbook-to-man.ts
@@ -60,7 +60,7 @@ GI:		REFENTRY
 #StartText:	^'\\" t^${_action 1001}
 StartText:	^${_action 1001}.TH "${_followrel descendant REFENTRYTITLE 1000}"
 		"${_followrel descendant MANVOLNUM 1000}"^
-EndText:	^.\\" created by instant / docbook-to-man, ${date}^
+EndText:	^.\\" created by instant / docbook-to-man^
 -
 #
 GI:		DOCINFO
